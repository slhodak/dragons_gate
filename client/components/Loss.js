import React from 'react';
import '../style.css';

export default (props) => {
  const { loss } = props;
  const active = loss > 0
  return (
    <span className={`loss ${active && 'disappearing'}`}>
      {active && ` -${loss}`}
    </span>
  )
}
