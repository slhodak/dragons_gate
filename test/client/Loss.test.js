import React from 'react';
import renderer from 'react-test-renderer';
import Loss from '../../client/components/Loss.js';

test('renders correctly', () => {
  const tree = renderer
    .create(<Loss loss={3} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
