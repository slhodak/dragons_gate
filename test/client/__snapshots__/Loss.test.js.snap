// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<span
  className="loss disappearing"
>
   -3
</span>
`;
